vello_encoding/src/path.rs

```diff
@@ -833,6 +902,174 @@ impl skrifa::outline::OutlinePen for PathEncoder<'_> {
         self.cubic_to(cx0, cy0, cx1, cy1, x, y);
     }
 
+    fn close(&mut self) {
+        self.close(false);
+    }
+}
+
+/// Encoder for path segments with winding information.
+pub struct WindingPathEncoder<'a> {
+    /// The underlying path encoder
+    encoder: PathEncoder<'a>,
+    path_windings: &'a mut Vec<f32>,
+    /// Collects points from all subpaths for a single winding calculation
+    all_glyph_points: Vec<Point>,
```

Comment:
It's not obvious to me why we're storing the points rather than accumulating the area on the fly.